src/main/resources/public/ts/models/badge-type.model.ts

```diff
@@ -43,14 +50,37 @@ export class BadgeType extends MinibadgeModel<BadgeType> {
         this.label = data.label;
         this.description = data.description;
         this.createdAt = data.createdAt;
+        this.level = data.level;
+        this.assignableBy = data.assignableBy;
         this.owner = new User(<IUserResponse>data.owner);
+        this.settings = data.settings;
         return this;
     }
 
     toModel(model: any): BadgeType {
         return new BadgeType(model)
     };
 
+    displayAssignors = (): string => {
+        let assigners = [];
+
+        this.settings.forEach(setting => {
+            setting.relations.forEach(relation => {
+                assigners.push(relation.assignorType.type + " ");
+            })
+        })
+        return assigners.toString()
```

Comment:
En prenant en compte le fait qu'on ne récupère qu'un seul setting (com' plus haut):

```suggestion
        this.setting.relations.map((relation) => relation.assignorType.type).join(", ")
```

pareil pour receiver